frontend/public/vendor.scss

```diff
@@ -40,7 +39,6 @@
 @import '~patternfly/dist/sass/patternfly/toolbar';
 @import '~patternfly/dist/sass/patternfly/blank-slate';
 @import '~patternfly/dist/sass/patternfly/list-view';
-@import '~patternfly/dist/sass/patternfly/popovers';
```

Comment:
Looks like this might still be used in kubevirt-web-ui-components

cc @mareklibra @vojtechszocs